Fix wording (#31191)
dotnet/docs: docs/core/porting/upgrade-assistant-uwp-framework.md

This release is currently in preview, and is receiving frequent updates. If you discover problems using the tool, report them in the tool's [GitHub repository](https://github.com/dotnet/upgrade-assistant). Use the **UWP** area tag so that all UWP related issues can be redirected to us.

> > [!NOTE]

Reviewer: ```suggestion
> [!TIP]
```

> > You may also refer to the UWP migration guide [here](/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/migrate-to-windows-app-sdk-ovw) that will tell you more about the changes from UWP APIs to the new Windows App SDK supported APIs and capabilities.  

Reviewer: ```suggestion
> You can also refer to the [UWP migration guide](/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/migrate-to-windows-app-sdk-ovw), which tells you more about the changes from UWP APIs to the new Windows App SDK-supported APIs and capabilities.  
```
